src/components/VolunteerFormular.vue

```diff
@@ -226,14 +226,15 @@ export default {
           setTimeout(() => {
             this.errorMessage = false
           }, 5000)
+        // stop fetching to avoid loop:
+         this.volunteerStoreStore.fetching = false
```

Comment:
this could not be right. If there is a timeout problem we have to handle it directly in the store or check the backend for errors.